app/models/claim.rb

```diff
@@ -345,6 +344,37 @@ def self.below_min_qa_threshold?
     (current_academic_year.approved.qa_required.count.to_f / claims_approved_so_far) * 100 <= MIN_QA_THRESHOLD
   end
 
+  # e.g. Claim.destroy_all_for_policy(MathsAndPhysics)
+  # Execute in a rails console
+  # Payments are deleted first due to HABTM (claims -< claim_payments >- payments)
+  # Also a payment might encompass 2 claims from multiple policies
+  def self.destroy_all_for_policy(policy)
```

Comment:
```suggestion
  def self.destroy_all_for_policy(policy)
    raise "claims for policy #{policy} cannot be destroyed" unless policy == MathsAndPhysics
```


From what I understand this is going to be used only once (for the foreseeing future); in that case I think we should prevent accidental usage of this method, otherwise it might be safer to move this method to a data migration to invoke once with `policy = MathsAndPhysics` as argument